Dockerfile

```diff
@@ -14,6 +14,7 @@ COPY webpack.server.js ./
 COPY .babelrc ./
 
 RUN npm install
+RUN npm install --save react-mdl
```

Comment:
This shouldn't be added to the Dockerfile. The `npm install --save` adds the dependency to the `package.json` file, which will be installed by and subsequent `npm install`s. 

Make sure that you have all changes of the `package.json` and `package-lock.json` committed.